gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -744,31 +744,32 @@ _message_handler (void *thread_data)
       (TensorQueryMsgThreadData *) thread_data;
   TensorQueryConnection *_conn = _thread_data->conn;
   TensorQueryServerData *_sdata = _thread_data->sdata;
-  TensorQueryCommandData cmd_data;
+  TensorQueryCommandData cmd_data_receive;
+  TensorQueryCommandData cmd_data_send;
   GstBuffer *outbuf = NULL;
   GstMetaQuery *meta_query;
 
-  cmd_data.cmd = _TENSOR_QUERY_CMD_CLIENT_ID;
-  cmd_data.client_id = g_get_monotonic_time ();
+  cmd_data_receive.cmd = _TENSOR_QUERY_CMD_CLIENT_ID;
+  cmd_data_receive.client_id = g_get_monotonic_time ();
```

Comment:
Send and receive command buffers are separated , so how about changing here also?
```suggestion
  cmd_data_send.cmd = _TENSOR_QUERY_CMD_CLIENT_ID;
  cmd_data_send.client_id = g_get_monotonic_time ();
```